Docs 9988 alerts reorg (#153)
mulesoft/docs-monitoring: modules/ROOT/pages/alerts.adoc

When you create basic and advanced alerts, you can select the severity for those alerts. The severity indicates how important the alert is for the recipients of the alerts.

[[alert_severity]]
Filter alerts based on severity using the following filters:

All severities::
Lists all severities

Not defined:: 
Lists alerts that do not have severities assigned

include::partial$include-alert-severity.adoc[leveloffset=+1]

=== Enabling and Disabling Alerts

You can enable or disable an alert by toggling the <<alerts, switch>> on the right:

* Enabling a disabled alert causes the alert to transition to `Pending` until it is time for evaluation. +
Depending on the value, the alert transitions to `OK` or `Alerting`.
* Disabling an alert causes the alert to transition to `Disabled`.

== Configure API Manager and Runtime Manager Alerts

From the *Alerts* page in Anypoint Monitoring, you can link to configuration pages for API Manager and Runtime Manager alerts.

Depending on your subscription level:

* If no alerts are set up, you might see links for API and runtime alert setup:
+
image::alerts-setup.png[API and Runtime Alert Setup, 75%, 75%]
+
** API operational alerts

> 

Author: ```suggestion
+
```

For guidance about alerts in API Manager, see xref:2.x@api-manager::using-api-alerts.adoc[About API Alerts]. Note that these alerts are not listed in the *Alerts* page for Anypoint Monitoring.
** Runtime Manager operational alerts

> 

Author: ```suggestion
+ 
```

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:page-aliases: alerts-using.adoc, alerts-config.adoc

Alerts indicate whether a resource (such as a Mule app) is behaving as expected or exceeding a defined threshold.

> You can implement any of several types of alerts. From the *Alerts* page in Anypoint Monitoring, you can create xref:basic-alerts.adoc[basic alerts], or you can link to alerts for API Manager and Runtime Manager alerts.

Author: ```suggestion
You can implement any of several types of alerts. From the *Alerts* page in Anypoint Monitoring, you can create xref:basic-alerts.adoc[basic alerts], or you can link to existing alerts for API Manager and Anypoint Runtime Manager.
```

If you have a Titanium subscription, you can also view, enable, and disable xref:advanced-alerts.adoc[advanced alerts] that are configured for graphs in custom dashboards.

> Using Anypoing Monitoring, you can configure two different types of alerts:

Author: lol :)

* xref:basic-alerts.adoc[Basic alerts] for servers and Mule apps
+
Limit per organization: Up to 50 basic alerts for users who do not have a Titanium subscription to Anypoint Platform. +
If you have a Titanium subscription, the limit is 100 basic alerts, and the limits increase as you purchase more vCores. For complete details about subscriptions, contact your customer support representative. See also https://www.mulesoft.com/anypoint-pricing[Anypoint Platform pricing].
* xref:advanced-alerts.adoc[Advanced alerts] for graphs in custom dashboards in Anypoint Monitoring
+
Limit per organization: Up to 20 advanced alerts.
+

> Note that thresholds are checked every 5 minutes. The evaluated query is the average value of the metric in the last 5 minutes. The evaluated metric is the one referenced in the *General* tab for the graph. If the value passes the threshold, the advanced alert is triggered. The advanced alert is triggered only if the alert state changes (from `OK` state to `Alerting` state or vice versa).

Author: ```suggestion
Note that thresholds are checked every five minutes. The evaluated query is the average value of the metric in the last five minutes. The evaluated metric is the one referenced in the *General* tab for the graph. If the value passes the threshold, the advanced alert is triggered. The advanced alert is triggered only if the alert state changes (from `OK` state to `Alerting` state or vice versa).
```

* Operational alerts for APIs through API Manager

> * Operational alerts for the runtime through Runtime Manager

Author: We can't reproduce limits for other products' alerts in case they change. As for merging limits, I will reevaluate that.

I think Mule runtime engine is right.

> * Operational alerts for APIs through API Manager
> * Operational alerts for the runtime through Runtime Manager

Author: ```suggestion
* Operational alerts for APIs through API Manager
* Operational alerts for server and app events through Runtime Manager
```

== Use the Alerts Page

> When you set up alerts, you can enable or disable any of them, filter alerts based on the <<alert_status, alert status>>, filter alerts based on the <<alert_severity, alert severity>>, and link to the graph in a custom dashboard that is associated with the alert.

Author: ```suggestion
After you set up alerts, you can enable or disable any of them, filter alerts based on the <<alert_status, alert status>>, filter alerts based on the <<alert_severity, alert severity>>, and link to the graph in a custom dashboard that is associated with the alert.
```

> When you set up alerts, you can enable or disable any of them, filter alerts based on the <<alert_status, alert status>>, filter alerts based on the <<alert_severity, alert severity>>, and link to the graph in a custom dashboard that is associated with the alert.

Author: It is a bit, but we do want to highlight some functionality and also emphasize that they don't have to delete alerts when they don't want to use them for a moment.

The *Alerts* page shows basic alerts for servers and apps or advanced alerts (https://www.mulesoft.com/anypoint-pricing[Titanium subscription required]) for graphs in custom dashboards.

[[alerts]]
image::alerts.png[Examples Alerts]

=== Filtering Alerts by Alert State

[[alert_status]]

> Filter alerts based on state using the following filters:

Author: This isn't too deep! No big deal. 
But I think it is obvious here. I think a screenshot of the panel would help them, as you mentioned above.

All states::
Lists all alerts.

include::partial$include-alert-state.adoc[leveloffset=+1]

=== Filtering Alerts by Alert Severity

When you create basic and advanced alerts, you can select the severity for those alerts. The severity indicates how important the alert is for the recipients of the alerts.

[[alert_severity]]
Filter alerts based on severity using the following filters:

> include::partial$include-alert-severity.adoc[leveloffset=+1]

Author: Added.

=== Enabling and Disabling Alerts

> You can activate or deactivate an alert by toggling the <<alerts, switch>> on the right.

Author: ```suggestion
You can enable or disable an alert by toggling the <<alerts, switch>> on the right:
```

> You can activate or deactivate an alert by toggling the <<alerts, switch>> on the right.

Author: Yes - this is incorrect language. Good catch.

> * Enabling a disabled alert causes the alert to transition to `Pending` until it is time for evaluation. +

Author: ```suggestion
* Enabling a disabled alert causes the alert to transition to `Pending` until it is time for evaluation. 
+
```

Depending on the value, the alert transitions to `OK` or `Alerting`.
* Disabling an alert causes the alert to transition to `Disabled`.

== Configure API Manager and Runtime Manager Alerts

From the *Alerts* page in Anypoint Monitoring, you can link to configuration pages for API Manager and Runtime Manager alerts.

Depending on your subscription level:

* If no alerts are set up, you might see links for API and Runtime alert setup, for example:
+
image::alerts-setup.png[API and Runtime Alert Setup, 75%, 75%]
+
** API Operational Alerts: For guidance with alerts in API Manager, see xref:2.x@api-manager::using-api-alerts.adoc[About API Alerts]. Note that these alerts are not listed in the Alerts page for Anypoint Monitoring.
** Runtime Operational Alerts: For guidance with alerts in Runtime Manager, see xref:runtime-manager::alerts-on-runtime-manager.adoc[About Configuring Alerts using Runtime Manager]. Note that these alerts are not listed in the *Alerts* page for Anypoint Monitoring.

> * *https://www.mulesoft.com/anypoint-pricing[Titanium Subscription Required]* - Instead of seeing links to API and Runtime alerts, you receive visual guidance showing how to open the configuration area for alerts in graphs for custom dashboards:

Author: No, it's not correct. Thank you for looking into that.

* Operational alerts for APIs through API Manager
* Operational alerts for server and app events through Runtime Manager

== Use the Alerts Page

After you set up alerts, you can enable or disable any of them, filter alerts based on the <<alert_status, alert status>>, filter alerts based on the <<alert_severity, alert severity>>, and link to the graph in a custom dashboard that is associated with the alert.

The *Alerts* page shows basic alerts for servers and apps or advanced alerts (https://www.mulesoft.com/anypoint-pricing[Titanium subscription required]) for graphs in custom dashboards.

[[alerts]]
image::alerts.png[Examples Alerts]

=== Filtering Alerts by Alert State

[[alert_status]]
Filter alerts based on state using the following filters:

All states::
Lists all alerts

include::partial$include-alert-state.adoc[leveloffset=+1]

=== Filtering Alerts by Alert Severity

When you create basic and advanced alerts, you can select the severity for those alerts. The severity indicates how important the alert is for the recipients of the alerts.

[[alert_severity]]
Filter alerts based on severity using the following filters:

> 

Author: ```suggestion

All severities::
Lists all severities

Not defined:: 
Lists alerts that do not have severities assigned

```
